DetectorDescription/DDCMS/interface/DDSolidShapes.h

```diff
@@ -57,6 +57,17 @@ namespace cms {
       }
       return std::begin(a)->value;
     }
+
+    template <class Mapping, class V>
+    typename Mapping::value_type::name_type value_reverse(Mapping a, V value) {
```

Comment:
that is actually a name you are asking for

Reply:
Renamed name_from_value (if I called it 'name' the compiler complained about ambiguity between this one and the `name` funcion that has a string as argument - perhaps I'm missing something)